boards/nucleo144-f207/include/periph_conf.h

```diff
@@ -258,16 +258,18 @@ static const spi_conf_t spi_config[] = {
 
 /**
  * @name    ADC configuration
- *
- * We need to define the following fields:
- * PIN, device (ADCx), channel
  * @{
  */
-#define ADC_CONFIG {              \
-    {GPIO_PIN(PORT_A, 3), 0, 3},  \
-    {GPIO_PIN(PORT_C, 0), 1, 0}  \
-}
-#define ADC_NUMOF          (2)
+static const adc_conf_t adc_config[] = {
```

Comment:
This configuration is the same for nucleo144-f412 and nucleo144-f413, maybe they could be put in a shared common file ? in `boards/nucleo144-common/include/periph-common144.h` or something ?

nucleo144-f429 and nucleo144-f446 have a different one.

Reply:
we could do that, but I think if it is only those two, it makes sense to keep them as is. Otherwise we start with many files that are only applicable to random sub-groups of boards?!

Reply:
agreed, let's keep it like this

Reply:
perfect.